### ISSUE TITLE
bug: footnotes show a linebreak before the indicator

### ISSUE THREAD

**Issue by davidsneighbour** [davidsneighbour/kollitsch.dev#188]
![20221020-22:05:25-Selection-001](https://user-images.githubusercontent.com/83281/196986526-209ba3ad-d6dd-418f-b22b-e3cd50fc6a89.png)

HTML:

```html
<sup id="fnref:1"><a href="#fn:1" class="footnote-ref" role="doc-noteref">1</a></sup>
```

The `a` inside the footnote indicator has a `:before` property that adds a `display: block` for some reasons.

**Comment by davidsneighbour** [comment]
Culprit:

```css
/**
 * fix navigation overflow for anchor links
 */
:target::before {
  display: block;
  height: $heading-height;
  margin: -$heading-height 0 0;

  content: '';
}
```

**Comment by davidsneighbour** [comment]
fixed in d592eb025152
